models/spring-ai-openai/src/test/java/org/springframework/ai/openai/chat/OpenAiChatModelIT.java

```diff
@@ -789,6 +789,21 @@ void streamWebSearchAnnotationsTest() {
 		logger.info("Full Content: {}", fullContent);
 	}
 
+	@Test
+	void testReasoningEffortWithDifferentLevels() {
```

Comment:
This test can be an overhead as this doesn't exactly verify the reasoning tokens from the output. What do you think?